src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -2071,7 +2119,9 @@ class NodeUtils(object):
             minor = self._discover_mig_minor(gpu_num, giid, ciid)
             if minor == -1:
                 continue
-            uuid = 'MIG-%s/%s/%s' % (gpus[gpuid]['uuid'], giid, ciid)
+
+            uuid = uuid_list[uuid_count]
```

Comment:
you would then pull the `GPU` and `Instance ID` columns and use that to access the `uuid_map`.